README.md

```diff
@@ -24,6 +24,11 @@ it is possible to build a series of packages with a single command. This is
 achieved by the "groups" defined in the `groups/` directory, with files defining
 a list of packages to build.
 
+Automation
+----------
+
+Under GitHub issues and pull requests, one can directly invoke package building on GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
```

Comment:
You may initiate package build tests via GitHub Actions.

_One does not simply use "one."_

Reply:
```suggestion
Under GitHub issues and pull requests, you may initiate package build tests via GitHub Actions, see [reference of workflows](.github/workflows/README.md) for details.
```